gradle.properties

```diff
@@ -0,0 +1 @@
+android.sdk.channel=2
```

Comment:
It must be as otherwise @k0shk0sh credentials will be pushed. it is only user for release.